### ISSUE TITLE
loss do not decrease...

### ISSUE THREAD

**Issue by guotong1988** [guotong1988/BERT-multi-gpu#5]

**Comment by guotong1988** [comment]
edit
```
optimizer = optimization_gpu.create_optimizer(
    None, FLAGS.learning_rate, FLAGS.num_train_steps, FLAGS.num_warmup_steps, False)
```
to
```       
optimizer = tf.train.AdamOptimizer(learning_rate=FLAGS.learning_rate)
```
